scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -58,12 +67,62 @@ trait BaseReplState {
     useHdfsMode_()
     customConfig -= mr1Key
     customConfig -= mr2Key
+    modeBanner
   }
 
   def useHdfsLocalMode() {
     useHdfsMode_()
     customConfig += mr1Key -> mrLocal
     customConfig += mr2Key -> mrLocal
+    modeBanner
+  }
+
+  def modeBanner: Unit = {
+    val scaldingMode = mode match {
+      case localMode: Local => {
+        (localMode.toString, System.getProperty("user.dir"))
+      }
+      case hdfsMode: Hdfs => {
+        val defaultFs = FileSystem.get(hdfsMode.jobConf)
+        val m = customConfig.get(mr2Key) match {
+          case Some("local") =>
+            s"${hdfsMode.getClass.getSimpleName}Local(${hdfsMode.strict})"
+          case _ =>
+            s"${hdfsMode.getClass.getSimpleName}(${hdfsMode.strict})"
+        }
+        (m, defaultFs.getWorkingDirectory.toString)
+      }
+    }
+    println(s"${Console.GREEN}#### Scalding mode: ${scaldingMode._1}")
+    println(s"#### User home: ${scaldingMode._2}${Console.RESET}")
+  }
+
+  /**
+   * List files under path
+   *
+   * @param pathStr
+   * @return
+   */
+  def fs_ls(pathStr: String) = {
```

Comment:
What's the return type here? Looks like Any to me. Can you put a type and make it `Try[FsShell]` I guess?